pkg/operator/ceph/cluster/controller.go

```diff
@@ -801,24 +753,22 @@ func (c *ClusterController) removeFinalizer(obj interface{}) {
 }
 
 // updateClusterStatus updates the status of the cluster custom resource, whether it is being updated or is completed
-func (c *ClusterController) updateClusterStatus(namespace, name string, state cephv1.ClusterState, message string) error {
-	logger.Infof("CephCluster %s status: %s", namespace, state)
+func (c *ClusterController) updateClusterStatus(namespace, name string, state cephv1.ClusterState, message string) {
+	logger.Infof("CephCluster %s status: %s. %s", namespace, state, message)
 
 	// get the most recent cluster CRD object
 	cluster, err := c.context.RookClientset.CephV1().CephClusters(namespace).Get(name, metav1.GetOptions{})
 	if err != nil {
-		return fmt.Errorf("failed to get cluster from namespace %s prior to updating its status: %+v", namespace, err)
+		logger.Errorf("failed to get cluster from namespace %s prior to updating its status: %+v", namespace, err)
```

Comment:
With these errors, we should include what the new status is so that if a users' cluster status can't be trusted, at least that information is available in the logs alongside the error.

Reply:
The previous line in the log (see line 757) will show the current status. It seems redundant if we add it here.

Reply:
As a developer I agree that it's redundant, but I think for users all the information we give to them in the error message itself empowers them. I think about how many times I've seen someone ask on slack how to fix some problem, and the only information given is a single error line. Users can be myopic and often don't look further than the first error they find. This would be especially helpful if debug logging is turned on, as the previous message might be lost in a sea of messages the `ceph` tool spits out.

Reply:
In principal I agree that more info in messages can help reduce confusion. Good error messages can make all the difference in a product. In this case I don't see the concern of getting lost in other debug messages from the previous line, but i'll add it anyway.